lib/usd/utils/CMakeLists.txt

```diff
@@ -84,3 +90,87 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
             DESTINATION ${CMAKE_INSTALL_PREFIX}/lib OPTIONAL
     )
 endif()
+
+# -----------------------------------------------------------------------------
+# sources
+# -----------------------------------------------------------------------------
+
+add_library(${MAYAUSDUTILS_PYTHON_LIBRARY_NAME} SHARED)
+
+target_sources(${MAYAUSDUTILS_PYTHON_LIBRARY_NAME}
+PRIVATE
+  module.cpp
+  wrapMayaTransformAPI.cpp
+)
+
+# -----------------------------------------------------------------------------
+# python init
+# -----------------------------------------------------------------------------
+
+set(PY_INIT_FILES
+  __init__.py
+)
+
+mayaUsd_copyFiles(${MAYAUSDUTILS_PYTHON_LIBRARY_NAME}
+  DESTINATION 
+    ${CMAKE_CURRENT_BINARY_DIR}/${TARGET_NAME}
+  FILES
+    ${PY_INIT_FILES}
+)
+
+# -----------------------------------------------------------------------------
+# compiler configuration
+# -----------------------------------------------------------------------------
+target_compile_definitions(${MAYAUSDUTILS_PYTHON_LIBRARY_NAME}
+  PRIVATE
+    MFB_PACKAGE_NAME=${TARGET_NAME}
+    MFB_ALT_PACKAGE_NAME=${TARGET_NAME}
+    MFB_PACKAGE_MODULE=${TARGET_NAME}
+)
+
+# -----------------------------------------------------------------------------
+# include directories
+# -----------------------------------------------------------------------------
+target_include_directories(${MAYAUSDUTILS_PYTHON_LIBRARY_NAME} 
+  PUBLIC
+    ${UTILS_INCLUDE_LOCATION} 
+    ${MAYAUSDUTILS_INCLUDE_LOCATION} 
+    ${USD_INCLUDE_DIR}
+    )
+
+# -----------------------------------------------------------------------------
+# link libraries
+# -----------------------------------------------------------------------------
+target_link_libraries(${MAYAUSDUTILS_PYTHON_LIBRARY_NAME}
+  PRIVATE
+    ${TARGET_NAME}
+    ${python_LIBRARIES}
+    usdGeom
```

Comment:
@robthebloke  Add `Boost::python` here now.   This the now proper target that really needs to link again python stuff.

Change `${python_LIBRARIES}`  to ${PYTHON_LIBRARIES}.  Make sure to do the same thing in bunch of other places where you added this.